src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx

```diff
@@ -0,0 +1,127 @@
+---
+subject: "Docs"
+releaseDate: '2024-12-13'
+version: 'version: November 29-December 13, 2024'
+---
+
+### New docs
+
+* [Logging with the iOS agent](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-remote-logging/) introduces enhanced logging capabilities in the iOS agent, including log level configuration and automatic log collection features.
```

Comment:
Change the link to` /docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-remote-logging/` from `https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-remote-logging/`

Reply:
Updated